keps/sig-api-machinery/0015-dry-run.md

```diff
@@ -14,7 +14,7 @@ approvers:
 editor: apelisse
 creation-date: 2018-06-21
 last-updated: 2018-06-21
```

Comment:
@apelisse Can you please update the `last-updated` date too?

Reply:
done

Reply:
Thank you :)